src/AppInstallerRepositoryCore/Rest/Schema/CommonRestConstants.h

```diff
@@ -14,6 +14,7 @@ namespace AppInstaller::Repository::Rest::Schema
     const Utility::Version Version_1_7_0{ "1.7.0" };
     const Utility::Version Version_1_9_0{ "1.9.0" };
     const Utility::Version Version_1_10_0{ "1.10.0" };
+    const Utility::Version Version_1_12_0{ "1.12.0" };

```

Comment:
I suppose I never realized how many unnecessary allocating statics we had laying around.  <grumbles about C# devs>